lib/matplotlib/collections.py

```diff
@@ -195,13 +218,32 @@ def get_datalim(self, transData):
             offsets = offsets.filled(np.nan)
             # get_path_collection_extents handles nan but not masked arrays
 
+        result = transforms.Bbox.null()
```

Comment:
I guess the conditionals below are more clearly written as
```python
# (don't define result here)
if len(paths) and len(offsets) and not self._offsetsNone:
    if transforms.contains_branch(transData):
        # one case.
        return ...
    else:
        # other case.
        return ...
else:
    return transforms.Bbox.null()
```

Reply:
Well, "other case" has to run if `self._offsetsNone` is `True` which it won't in the above...

Reply:
then perhaps
```python
if len(paths) and len(offsets):
    if contains(transData):
        # other case
        return ...
    if not self._offsetsNone:
        # first case
        return ...
return null()
```
?